src/schema.ts

```diff
@@ -0,0 +1,56 @@
+//GraphQl Schema
+//para colocar input (input NomeInput)
+export const typeDefs = `#graphql
+
+  type Query{
+    hello: String
+  }
+
+  #input obrigatório !
+  type Mutation{
+    createUser(input: CreateUserInput!): User
+    login(input: LoginInput!): Login
```

Comment:
esses campos sempre são retornados, certo?
```suggestion
    createUser(input: CreateUserInput!): User!
    login(input: LoginInput!): Login!
```